faunadb/objects.py

```diff
@@ -58,6 +58,7 @@ def __eq__(self, other):
     return isinstance(other, Ref) and self.value == other.value
 
   def __ne__(self, other):
+    # pylint: disable=unneeded-not
```

Comment:
This doesn't appear to be necessary. When I run pylint I get: `E: 61, 0: Bad option value 'unneeded-not' (bad-option-value)`